pkg/driver/controller.go

```diff
@@ -155,15 +155,19 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		}
 	}
 
-	if config.GlobalConfig.EnableControllerSetQuota == nil || *config.GlobalConfig.EnableControllerSetQuota {
-		if util.SupportQuotaPathCreate(true, config.BuiltinCeVersion) && util.SupportQuotaPathCreate(false, config.BuiltinEeVersion) {
-			volCtx[common.ControllerQuotaSetKey] = "true"
-			d.quotaPool.Run(context.Background(), func(ctx context.Context) {
-				if err := d.setQuotaInController(ctx, volumeId, req.GetCapacityRange(), options, subPath, secrets, volCtx); err != nil {
-					log.Error(err, "set quota in controller error")
-				}
-			})
+	if config.GlobalConfig.EnableSetQuota == nil || *config.GlobalConfig.EnableSetQuota {
+		if config.GlobalConfig.EnableControllerSetQuota == nil || *config.GlobalConfig.EnableControllerSetQuota {
+			if util.SupportQuotaPathCreate(true, config.BuiltinCeVersion) && util.SupportQuotaPathCreate(false, config.BuiltinEeVersion) {
+				volCtx[common.ControllerQuotaSetKey] = "true"
+				d.quotaPool.Run(context.Background(), func(ctx context.Context) {
+					if err := d.setQuotaInController(ctx, volumeId, req.GetCapacityRange(), options, subPath, secrets, volCtx); err != nil {
+						log.Error(err, "set quota in controller error")
+					}
+				})
+			}
 		}
+	} else {
+		volCtx[common.DisableQuotaSetKey] = "true"
```

Comment:
once this ctx is added, it cannot be changed. If `EnableSetQuota` config changed to true, the quota cannot be set again.

I suggest not adding this ctx, and only checking the config in [node](https://github.com/juicedata/juicefs-csi-driver/pull/1406/files#diff-4f4d9c2e0ec5c4d3b4ac2d79abd70ee21bacff65ba05c059396921996dbf6607R198-R199).